website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -37,9 +37,15 @@ cd $HOME/projects/memcached-operator
 # so all API groups will be <group>.example.com
 operator-sdk init --domain example.com --repo github.com/example/memcached-operator
 ```
+API groups are are a mechanism to group portions of the Kubernetes API. You're probably already familiar with
```

Comment:
API groups are a mechanism to group portions of the Kubernetes API. You're probably already familiar with

Reply:
WDYT about nere we just link the doc: https://book.kubebuilder.io/cronjob-tutorial/gvks.html
Suggestion:

**Note** If you are not familiarized with Kubebernets APIs please ensure that you check [Groups and Versions and Kinds, oh my!](https://book.kubebuilder.io/cronjob-tutorial/gvks.html).

Reply:
fixed